### ISSUE TITLE
Locator and PAS expect a my.cnf file for the deploy user

### ISSUE THREAD

**Issue by carolyncole** [pulibrary/princeton_ansible#3074]
When transitioning to the mysql role the new role does not include a .my.cnf for the deploy user on the machine.  This is being utilized by capistrano in locator and PAS

**Comment by carolyncole** [comment]
```
ASYNC FAILED on maria-staging2.princeton.edu: jid=594265136296.32669
failed: [maria-staging2.princeton.edu] (item={'name': 'locator_staging', 'encoding': 'utf8mb4', 'collation': 'utf8mb4_general_ci'}) => {"ansible_job_id": "594265136296.32669", "ansible_loop_var": "item", "changed": false, "finished": 1, "item": {"collation": "utf8mb4_general_ci", "encoding": "utf8mb4", "name": "locator_staging"}, "msg": "Cannot dump database(s) 'locator_staging' - not found", "results_file": "/root/.ansible_async/594265136296.32669", "started": 1, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```

**Comment by kayiwa** [comment]
> ```
> ASYNC FAILED on maria-staging2.princeton.edu: jid=594265136296.32669
> failed: [maria-staging2.princeton.edu] (item={'name': 'locator_staging', 'encoding': 'utf8mb4', 'collation': 'utf8mb4_general_ci'}) => {"ansible_job_id": "594265136296.32669", "ansible_loop_var": "item", "changed": false, "finished": 1, "item": {"collation": "utf8mb4_general_ci", "encoding": "utf8mb4", "name": "locator_staging"}, "msg": "Cannot dump database(s) 'locator_staging' - not found", "results_file": "/root/.ansible_async/594265136296.32669", "started": 1, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
> ```

this works "as designed" No such database exists. :-)

**Comment by kayiwa** [comment]
Closing this. We moved to `mysql-db-staging1`
